classes/Utils.php

```diff
@@ -902,7 +902,7 @@ public function get_course_completed_percent( $course_id = 0, $user_id = 0, $get
 					 */
 					$is_completed = apply_filters( 'tutor_is_zoom_lesson_done', false, $content->ID, $user_id );
 					if ( $is_completed ) {
-						$completed_count++;
```

Comment:
Why this change? Please make sure it will not affect any other areas!

Reply:
I need to check these. My ide formatter might formatted this. I'll check and fix this